### ISSUE TITLE
v6 Conf/TopicConf types problem

### ISSUE THREAD

**Issue by b1rdex** [php-rdkafka/php-rdkafka#501]
### Description

The following code:

```php
<?php
$rdKafkaTopicConfig = new \RdKafka\TopicConf();
$producerTopic = $this->producerConfig->getTopicConfig($envelope);
foreach ($producerTopic->getParams() as $name => $value) {
    $rdKafkaTopicConfig->set($name, $value);
}
$topic = $producer->newTopic($producerTopic->getName(), $rdKafkaTopicConfig);
```

But I expected this output instead:
```
 Argument #2 ($topic_conf) must be of type ?RdKafka\Conf, RdKafka\TopicConf given
```

### php-rdkafka Version

6.0.0RC2

### librdkafka Version

1.6.0

### PHP Version

8.0

### Operating System

debian 11 / alpine

**Comment by b1rdex** [comment]
v5 code works fine, v6 = fatal error. IMO the type is incorrect and should be reverted.

**Comment by b1rdex** [comment]
Here is a snippet from the readme:
![image](https://user-images.githubusercontent.com/312855/144968106-1dcd25d2-eac6-4404-831d-baa7d4812891.png)

Looks like a test for this part is missing?

**Comment by b1rdex** [comment]
https://github.com/arnaud-lb/php-rdkafka/blob/c4260fb5354460cb1d96671607f50c2f23753054/rdkafka.stub.php#L40

https://github.com/arnaud-lb/php-rdkafka/blob/c4260fb5354460cb1d96671607f50c2f23753054/kafka_consumer.stub.php#L52

**Comment by arnaud-lb** [comment]
Thank you @b1rdex

Fixed in #502
